frontend/src/locales/en.yml

```diff
@@ -185,6 +185,40 @@ quick-start:
   notice:
     title: Quickstart guide dismissed
     desc: You can still bring it back later from the top right menu
+auth:
+  common:
+    sign-in: Sign in
+    sign-up: Sign up
+    email: Email
+    name: Name
+    password: Password
+    activate: Activate
```

Comment:
These words looks like the 'common words'. we can extract these word into the 'common' scope.

Reply:
done, PTAL